### DISCUSSION TITLE
Branch 이름에 대한 규칙추가

### DISCUSSION THREAD

**Discussion by moaoh5113** [God-save-the-carrots/ft_transcendence#58] [Ideas]
저희가 `develop` 브랜치 아래에서 모든 파트가 한번에 작업을 하다보니 작업을 진행하다보면 브랜치명이 겹치거나 어떠한 파트에서 작업을 했는지 알아보기 힘들 거 같다라는 생각이 들게되었습니다. 
그래서 브랜치 이름에 간단하게 나마 규칙을 추가해서 브랜치명을 짓는거는 어떻게 생각하시는지 의견을 나누어보고싶습니다.
일단 고민해본 저의 생각으로는 `<issue>/<part>-<content>` 이와 같은 형식으로 규칙을 정하여 
예를 들어 **backend** 파트에서 **DB**를 **추가**하는 작업을 진행하는 브랜치라고 하면 `feat/be-add-db-table` 이와 같은 형식으로 
이름을 지어두면 조금 더 알아보기 쉬울 거 같다는 생각이 들어 제안을 해 봅니다.

**`frontend - fe`**
**`backend - be`**
**`game - game?`**

**Comment by sindydwns** [comment]
찬성합니다. 그러나 명칭은 조금 바뀌었으면 합니다.
`frontend - fe` -> `frontend - front`
`backend - be` -> `backend - back`
저는 축약어를 음성으로 읽었을때 의미를 바로 파악할 수 있는 단어가 좋습니다.

그래도 짧은게 좋으시다면..
`frontend - fe` -> `frontend - fr`
`backend - be` -> `backend - bk`
`game - game` -> `game - gm`

**Comment by moaoh5113** [comment]
01.22 회의에서 해당 내용에 대해 이야기를 나누었고,
해당 안건이 긍정적으로 이야기가 되어
`frontend - FE`
`backend - BE`
`game - GM`
`devops - OS`
으로 하여 `ex) feat/FE-create-spa` 으로 위와같은 형식으로 진행을 하기로 하였습니다.
